### PR TITLE
docs(readme): update dockerImage

### PR DESCRIPTION
**Pull request by JamesSingleton**

### CODE REVIEW

File: packages/one-app-runner/README.md [hunk 1]
```diff
@@ -34,7 +34,7 @@ npm install --save-dev @americanexpress/one-app-runner
     "modules": ["."],
     "rootModuleName": "frank-lloyd-root",
     "moduleMapUrl": "https://example.com/cdn/module-map.json",
-    "dockerImage": "one-app-dev:5.0.0",
+    "dockerImage": "oneamex/one-app-dev:latest",
```
Comment (infoxicator) [inline packages/one-app-runner/README.md:37]: @JamesSingleton here we use latest but all the other examples below use 5.0.0? latest should be the recommended version

Comment (JAdshead) [reply packages/one-app-runner/README.md]: Ideally we should use latest right ?

Comment (JamesSingleton) [reply packages/one-app-runner/README.md]: I just took this as more of an example. What if someone wants to use a specific version of One App?

Comment (JAdshead) [reply packages/one-app-runner/README.md]: thats fair, maybe use the most recent version

**Pull request merged**